scaladoc/test/dotty/tools/scaladoc/signatures/AbstractMemberSignaturesTest.scala

```diff
@@ -26,7 +26,7 @@ class AbstractMembers extends ScaladocTest("abstractmembersignatures"):
   }
 
   private def signaturesFromDocumentation()(using DocContext): Map[String, List[(String, String)]] =
-    val output = summon[DocContext].args.output.toPath.resolve("api")
+    val output = summon[DocContext].args.output.toPath
```

Comment:
Should this still use the `api` subdirectory in case of `-Yapi-subdirectory`?

Reply:
Oh, it’s in the tests, so forget about my comment.

Reply:
this is onlu test and I do not implement version for `-Yapi-subdirectory` since this tests are focused on signature not layout of repo. I've added basic test for structure with `-Yapi-subdirectory`